src/html2ans/parsers/text.py

```diff
@@ -277,30 +303,49 @@ class ListParser(BaseElementParser):
     .. code-block:: python
 
         {
-            "type": "list",
-            "list_type": "unordered",
-            "items": [
+            'type': 'list',
+            'list_type': 'unordered',
+            'items': [
                 {
-                    "type": "text",
-                    "content": "Post Reports is the daily podcast from The Washington Post."
+                    'type': 'text',
```

Comment:
This is outdated now, right?

Reply:
Ah, it's not. That's a problem. The first list element should not get split into 3 parts